lib/modules/modules.go

```diff
@@ -101,6 +101,14 @@ type EntitlementInfo struct {
 	Limit int32
 }
 
+// WithinLimit tests that a given entitlement is within its prescribed limit
+// based on the supplied use count. The actual definition of a "use" depends on
+// the entitlement in question. A disabled entitlement is always out of its
+// limit.
+func (e EntitlementInfo) WithinLimit(count int) bool {
+	return e.Enabled && (e.Limit == 0 || count <= int(e.Limit))
+}
```

Comment:
this is awesome 👍 but i'm not so sure about this

```
count <= int(e.Limit))
```
if the count is `equal` to limit, wouldn't that means you are at your limit?

your test name also suggests it too `at limit`:
```
		{
			name:   "at limits returns true",
			count:  10000,
			uut:    modules.EntitlementInfo{Enabled: true, Limit: 10000},
			assert: require.True,
		},
```

Reply:
> if the count is equal to limit, wouldn't that means you are at your limit?

That should be fine if you have  the  `e.Limit` set to 1 you should be able to create 1 access list

Reply:
Explicitly renamed to `UnderLimit()` and made the range check exclusive.